test/suites/optional/multus.robot

```diff
@@ -0,0 +1,151 @@
+*** Settings ***
+Documentation       Tests for Multus and Bridge plugin on MicroShift
+
+Resource            ../../resources/common.resource
+Resource            ../../resources/microshift-process.resource
+
+Suite Setup         Setup
+Suite Teardown      Teardown
+
+
+*** Variables ***
+${BRIDGE_INTERFACE}                 br-test
+${BRIDGE_NAD_YAML}                  ./assets/multus/bridge-nad.yaml
+${BRIDGE_POD_YAML}                  ./assets/multus/bridge-pod.yaml
+${BRIDGE_POD_NAME}                  test-bridge
+${BRIDGE_IP}                        10.10.0.10/24
+
+${PREEXISTING_BRIDGE_INTERFACE}     br-preexisting
+${PREEXISTING_BRIDGE_NAD_YAML}      ./assets/multus/bridge-preexisting-nad.yaml
+${PREEXISTING_BRIDGE_POD_YAML}      ./assets/multus/bridge-preexisting-pod.yaml
+${PREEXISTING_BRIDGE_POD_NAME}      test-bridge-preexisting
+${PREEXISTING_BRIDGE_IP}            10.10.1.10/24
+
+${NS}                               default
+
+
+*** Test Cases ***
+Pre-Existing Bridge Interface
+    [Documentation]    Test verifies if Bridge CNI plugin will work correctly with pre-existing interface.
+    [Setup]    Run Keywords
```

Comment:
You don't need to change it, but I'm curious why there are so many individual keywords in the setup for this case. Why not include them in the test body, or write a separate keyword?

Reply:
> Why not include them in the test body

I was being literal - test setup in `[Setup]`

> or write a separate keyword?

I didn't like how this requires a jump to different location in the file - this way everything is single place.

So I think I just felt like it :D It's not perfect though, this keyword (in this or future PR) is maxing out the length, but separate keyword was "context jump" and writing general function for both Bridge tests was quite ugly and required some mental processing on what happens with which arguments.

Reply:
It can definitely be hard to find a balance between inline and separate code blocks. That max keyword aspect to the linter is a nudge towards what the linter author thought we should be doing, but there have definitely been cases where we overrode the rule. Maybe we should have written more of those cases in Python? I don't know. As I said, there's no need to change anything here, I was just curious about the evolution of the code.

Reply:
> Maybe we should have written more of those cases in Python?

Perhaps, but python function don't support so great support for log.html. We'll see in the future